crates/evm/fuzz/src/strategies/param.rs

```diff
@@ -224,6 +263,332 @@ pub fn fuzz_param_from_state(
     }
 }
 
+/// Mutates the current value of the given parameter type and value.
+pub fn mutate_param_value(
+    param: &DynSolType,
+    value: DynSolValue,
+    test_runner: &mut TestRunner,
+    state: &EvmFuzzState,
+) -> DynSolValue {
+    let new_value = |param: &DynSolType, test_runner: &mut TestRunner| {
```

Comment:
there are a lot of calls to this `new_value`, i think we can have an inner function that returns `Option<_>` with the same implementation, just reducing the repetition of `new_value()`

Reply:
simplified in https://github.com/foundry-rs/foundry/pull/11061/commits/bc3afd397c673a3b1af4bc1a3c5a05ba7c7147e0 hope it makes sense